### DISCUSSION TITLE
Ordered list numbering limit

### DISCUSSION THREAD

**Discussion by aniketgm** [nvim-neorg/neorg#655] [Q&A]
First of all thanks a lot for this amazing neovim plugin. Kudos to the creators and developers.
I wanted to know, Is there a limit to the numbering of ordered list ? As shown in the image below. After '20' I don't see the numbering. Also why do the numbers have small font ?

![neorg-ordered-list-numbering-limit](https://user-images.githubusercontent.com/14215698/200760816-1a629d04-c60a-4c9f-83d0-47e00c8a6537.png)

**Comment by max397574** [comment]
the limit is 20 because that's how many unicode symbols there are
unicode symbols are used because the numbers can only be one symbol long because that's how long the text that's concealed (`~`) was
this is a limitation until we have anticonceal

**Reply by aniketgm** [reply]
Why do numbers have to be one symbol ? Why the restriction ? I mean I can write, 1.  -- \<one>\<dot> and the norg mode should understand this and then when I press enter and goto next line, it should put 2. automatically, no? With same level/space as 1.
Similarly for other signifiers: a A 1) a) A)

**Reply by max397574** [reply]
1. about the automatically detect things
no that's not a functionality yet
2. why one symbol
the `~` symbols are displayed as a number
it is atm impossible (neovim/neovim limitation) to display something with a longer string than it actually is (that would be anticonceal which isn't a thing yet)

**Reply by aniketgm** [reply]
Oh ok. (thanks for the quick response🙂)
I would suggest, maybe have a look at how Vimwiki does it.
It automatically detects and puts `2.` at the next line when user press \<enter>
